docs/Upgrading.md

```diff
@@ -36,6 +38,12 @@ We also recommend the use of a staging site which matches your production enviro
 
 If you do run into issues, we recommend looking at our [debugging tips.](https://github.com/Shopify/shopify_app/blob/main/docs/Troubleshooting.md#debugging-tips)
 
+## Unreleased
+The `Itp` controller concern has been removed from `LoginProtection`.
+If any of your controllers are dependant on methods from `Itp` then you can include `ShopifyApp::Itp` directly.
+You may notice a deprecation notice saying, `Itp will be removed in an upcoming version`.
+This is because we intend on removing `Itp` completely in `v22.0.0`, but this will work in the meantime.
+
```

Comment:
I hope this is the right place?

I was also thinking of adding the names of the methods like `set_test_cookie`, `set_top_level_oauth_cookie`, `clear_top_level_oauth_cookie`, ....

So it would be easy to search for that method being removed in the upgrading doc. How do you feel about that?